src/main/mule/coach.xml

```diff
@@ -662,11 +732,25 @@ output application/json
 		<logger
 			level="INFO"
 			doc:name="Log Stored URI Params"
-			doc:id="f5fb5455-22c7-4d1f-b316-fa26806f3f1b"
+			doc:id="5776800c-73ac-4089-a7f5-963b32492dbc"
 			message="URI Param - coachId: #[vars.coachId], teamseasonId: #[vars.teamSeasonId], sessionId: #[vars.sessionId] - stored as vars" />
+  
+
+
+  <flow-ref doc:name="secondaryFlow" name="get:\sessions\(sessionId)\assessments:salesforce-data-api-config" target="sessionAssessments"/>
```

Comment:
What's this flow for? @ZombieWoofers

Reply:
@archana-llamas this is the approach we used to include session assessment data in the attendance response, we originally tried within the salesforce query but weren't successful.